### PR TITLE
add `faster` flag to `./mach test` command 

### LINKED ISSUE #9253: `./mach test` returns error
The returned error:
```text
Error running mach:

    ['test']

The error occurred in code that was called by the mach command. This is either
a bug in the called code itself or in the way that mach is calling it.

You should consider filing a bug for this issue.

If filing a bug, please include the full output of mach, including this error
message.

The details of the failure are as follows:

TypeError: test_tidy() takes exactly 2 arguments (1 given)

  File "/home/alexander/dev/rust/servo/python/servo/testing_commands.py", line 145, in test
    Registrar.dispatch("test-%s" % suite, context=self.context, **kwargs)
  File "/home/alexander/dev/rust/servo/python/mach/mach/registrar.py", line 115, in dispatch
    return self._run_command_handler(handler, context=context, **kwargs)
  File "/home/alexander/dev/rust/servo/python/mach/mach/registrar.py", line 90, in _run_command_handler
    result = fn(**kwargs)

```

### PR DESCRIPTION
**Pull request by apopiak**
in order to call `test-tidy` properly

close #9253

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9273)

### CODE REVIEW

**Comment by apopiak** [comment]
`./mach test` returns the following error:
```
./python/servo/testing_commands.py:98: E128 continuation line under-indented for visual indent
```
which seems false to me. (referencing [this line](https://github.com/servo/servo/pull/9273/files#diff-a139f7c2483c6cf4f78098b575503c04R98))

File: python/servo/testing_commands.py [hunk 1]
```diff
@@ -90,9 +90,12 @@ def run_test(self, prefix, args=[], release=False):
...
     @CommandArgument('--release', default=False, action="store_true",
                      help="Run with a release build of servo")
-    def test(self, params, render_mode=DEFAULT_RENDER_MODE, release=False):
+    @CommandArgument('--faster', default=False, action="store_true",
+                     help="Only check changed files and skip the WPT lint")
+    def test(self, params, render_mode=DEFAULT_RENDER_MODE, release=False, faster=False):
         suites = OrderedDict([
-            ("tidy", {}),
+            ("tidy", {"kwargs": {"faster": faster},
+                     "include_arg": "include"}),
```
Comment (frewsxcv) [inline python/servo/testing_commands.py]: Format it like this:

```diff
-            ("tidy", {}),
+            ("tidy", {"kwargs": {"faster": faster},
+                      "include_arg": "include"}),
```

Comment (apopiak) [inline python/servo/testing_commands.py]: sry @frewsxcv , I don't see the difference o.O

Comment (apopiak) [inline python/servo/testing_commands.py]: OMG, 1 whitespace! How are you supposed to see this!

Comment (frewsxcv) [inline python/servo/testing_commands.py]: The spacing of the first character on the second line. The `"` should match up with the `"` above it

Comment (apopiak) [inline python/servo/testing_commands.py]: thx for the help :-)

Comment (frewsxcv) [inline python/servo/testing_commands.py]: It is pretty picky, though this standardized in [the official style guide for Python code](https://www.python.org/dev/peps/pep-0008/#indentation)

**Comment by Wafflespeanut** [comment]
Squash?

**Comment by apopiak** [comment]
sure

**Comment by apopiak** [comment]
@Wafflespeanut done :-)

**Pull request merged**